js/src/widgets/contextControls.js

```diff
@@ -200,10 +200,7 @@
       pickerOffset=pickerContainer.offset(),
       windowWidth = this.state.windowsElements[this.windowId].width();
       if (pickerContainer.width() + pickerOffset.left > windowWidth) {
-        pickerContainer.offset({
-          top: pickerOffset.top,
-          left: windowWidth - (pickerContainer.width())
-        });
+        pickerContainer.css('left', windowWidth - (pickerContainer.outerWidth()));
```

Comment:
I'm not sure this is ultimately the best way to position this, but I believe this is doing what the code was initially intending to do when written.